notebooks-html/linear_multiple.html

```diff
@@ -505,35 +507,36 @@ <h2 id="Multiple-Linear-Regression">Multiple Linear Regression<a class="anchor-l
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <p>Again, keep in mind that the actual matrix $ X $ only contains the numerical values of the table above.</p>
-<p>Notice that $ X $ is composed of multiple $ x $ vectors stacked on top of each other. To keep the notation clear, we define $ X_{i} $ to refer to the row with index $ i $ of $ X $. We define $ X_{ij} $ to refer to the element with index $ j $ of the row with index $ i $ of $ X $. Thus, $ X_i $ is a $ p $-dimensional vector and $ X_{ij} $ is a scalar. $ X $ is an $ n \times p $ matrix.</p>
-<p>For example, from the table above we have $ X_4 = [1, 140, 3449, 70] $ and $ X_{41} = 140 $. This notation becomes important when we define the cost function since we will need both $ X $, the matrix of input values, and $ y $, the vector of MPGs.</p></div></div></div>
+<p>Notice that $ X $ is composed of multiple $ x $ vectors stacked on top of each other. To keep the notation clear, we define $ X_{i} $ to refer to the row with index $ i $ of $ X $. We define $ X_{i,j} $ to refer to the element with index $ j $ of the row with index $ i $ of $ X $. Thus, $ X_i $ is a $ p $-dimensional vector and $ X_{i,j} $ is a scalar. $ X $ is an $ n \times p $ matrix, where $n$ is the number of car examples we have and $p$ is the number of attributes we have for a single car.</p>
```

Comment:
I like this change! Especially since it resolves the ambiguity when a matrix contains more than 10 rows.